tests/__mocks__/apex/data/recurringDataAccountResponse.json

```diff
@@ -0,0 +1,6 @@
+{
+  "DonorType": "Account",
```

Comment:
I see that we include these mock responses twice in different directories, is that required for these tests?

Reply:
I think so. The only alternative I was able to come up with would be moving it up the directory structure, something like `require(../../../data/recurringDataAccountResponse);`